src/storage/StorageServer.cpp

```diff
@@ -87,6 +88,11 @@ bool StorageServer::initWebService() {
     router.get("/admin").handler([this](web::PathParams&&) {
         return new storage::StorageHttpAdminHandler(schemaMan_.get(), kvstore_.get());
     });
+    router.get("/sysinfo").handler([this](web::PathParams&&) {
+        auto handler = new storage::SysInfoHandler();
+        handler->init(dataPaths_[0]);
```

Comment:
Just the first data path?

Currently, storaged support multi data paths.  So if you want to detect the disk usage,  all dataPaths should be handled.

Reply:
good points